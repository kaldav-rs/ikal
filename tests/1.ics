BEGIN:VCALENDAR
PRODID:-//Nextcloud calendar v1.5.0
VERSION:2.0
CALSCALE:GREGORIAN
BEGIN:VEVENT
CREATED:20170209T192358
DTSTAMP:20170209T192358
LAST-MODIFIED:20170209T192358
UID:5UILHLI7RI6K2IDRAQX7O
SUMMARY:Vers
CLASS:PUBLIC
STATUS:CONFIRMED
DTSTART;VALUE=DATE-TIME;TZID=Europe/Paris:20150219T190000
DTEND;VALUE=DATE:20170210
END:VEVENT
END:VCALENDAR
