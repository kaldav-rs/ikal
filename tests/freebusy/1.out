Ok(
    VFreebusy {
        dtstamp: Local(
            1997-09-01T10:30:00+02:00,
        ),
        uid: "19970901T082949Z-FA43EF@example.com",
        contact: None,
        dtstart: DateTime(
            Local(
                1997-10-15T07:00:00+02:00,
            ),
        ),
        dtend: Some(
            DateTime(
                Local(
                    1997-10-16T07:00:00+02:00,
                ),
            ),
        ),
        organizer: Some(
            "mailto:jane_doe@example.com",
        ),
        url: None,
        attendee: [
            "mailto:john_public@example.com",
        ],
        comment: [],
        freebusy: [],
        rstatus: [],
        x_prop: {},
        iana_prop: {},
    },
)
